contracts/src/IRollup.sol

```diff
@@ -308,10 +314,10 @@ interface IRollup {
      *
      * Emits: `AssertionCreated` and `StakerStaked` events.
      *
-     * @param vmHash New VM hash.
-     * @param inboxSize Size of inbox corresponding to assertion (number of transactions).
+     * @param blockNum Block number this assertion advances to.
+     * @param assertionData Versioned assertion data
```

Comment:
Can you leave a little bit of detail on what the structure of the data is. Maybe a ref somewhere or a "version int8 | vmHash int256" (assuming that's the structure). Maybe it's not terribly important as the data doesn't get used.